### ISSUE TITLE
Provide better support for IndieWeb initatives

### ISSUE THREAD

**Issue by errbufferoverfl** [errbufferoverfl/garden.errbufferoverfl.me#2]
tl;dr is here: https://indieweb.org/Getting_Started

# Identity
- [ ] h-card for homepage and icon
- [x] `rel-me` for identity-elsewhere and RelMeAuth sign in. Introduced in commit [#9526df8](https://github.com/errbufferoverfl/garden.errbufferoverfl.me/commit/9526df8710ef33cf5eef718fe1e5763c2636196e)

# Posts
- [ ] Add [h-feed](https://indieweb.org/h-feed), [h-entry](https://indieweb.org/h-entry) and [h-card](https://indieweb.org/h-card) support.

# Citability

- [ ] Permalinks, supported but need to be better exposed in UI.
- [x] [permashortcitations](https://indieweb.org/permashortcitations), too much effort for not enough payoff, won't do right now.
- [x] [permashortcitations](https://indieweb.org/permashortcitations), too much effort for not enough payoff, won't do right now.

# Syndication

- [ ] [POSSE](https://indieweb.org/POSSE) has some merit to it, but looks like a lot of work to add to Hugo. TBA.
- [ ] See `h-entry` on Posts

# Mentions

- [ ] [Webmentions](https://indieweb.org/webmention)

# Link Preview

- [x] Supported by Open Graph
